bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java

```diff
@@ -258,6 +290,8 @@ public Injector initialize()
         moduleList.add(Binder::requireExplicitBindings);
         moduleList.add(Binder::requireExactBindingAnnotations);
 
+        moduleList.add(binder -> binder.bind(ConfigurationResolver.class)
+                .toInstance(requireNonNull(configurationResolver, "configurationResolver is null")));
```

Comment:
See above. We shouldn't check for null here.

Reply:
I missed an initialization there -